File: sandbox/node/src/AccountCreateOauth.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.AccountApi();

const accountCreateRequest = new models.AccountCreateRequest();
accountCreateRequest.emailAddress = "newuser@dropboxsign.com";
accountCreateRequest.clientId = "cc91c61d00f8bb2ece1428035716b";
accountCreateRequest.clientSecret = "1d14434088507ffa390e6f5528465";

apiCaller.accountCreate(
    accountCreateRequest,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling Account#accountCreate:");
  console.log(error.body);
});

File: sandbox/dotnet/src/Dropbox.SignSandbox/AccountCreateOauth.cs
using System;
using System.Collections.Generic;
using System.IO;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class AccountCreateOauth
{
    public static void Run()
    {
        var config = new Configuration();

        var accountCreateRequest = new AccountCreateRequest(
            emailAddress: "newuser@dropboxsign.com",
            clientId: "cc91c61d00f8bb2ece1428035716b",
            clientSecret: "1d14434088507ffa390e6f5528465"
        );

        try
        {
            var response = new AccountApi(config).AccountCreate(
                accountCreateRequest: accountCreateRequest
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Account#AccountCreate: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
